qiling/os/windows/variables.py

```diff
@@ -0,0 +1,22 @@
+from qiling.os.windows.const import *
+
+# Setted for Windows 10 64bit desktop
+OS_VERSION_INFO = {"dwOSVersionInfoSize": 0xADD,
+                   VER_MAJORVERSION: 0xADD,
+                   VER_MINORVERSION: 0xADD,
+                   VER_BUILDNUMBER: 0xADD,
+                   VER_PLATFORMID: 0xADD,
+                   "szCSDVersion": 0xADD,
+                   VER_SERVICEPACKMAJOR: 0,
+                   VER_SERVICEPACKMINOR: 0xADD,
+                   VER_SUITENAME: 0xADD,
+                   VER_PRODUCT_TYPE: 0xADD,
+                   "wReserved": 0xADD,
+                   "OS": "1001"
+                   }
+
+# Environment
+Environment = {"windir": "C:\\Windows",
```

Comment:
sounds like a malware sandbox IDable name, but again if we reach malware can ID us, we are very famous by then